src/states/Game.gd

```diff
@@ -33,6 +33,13 @@ func _ready() -> void:
 			level.add_child(player)
 
 	reset_move_counter()
+#	hide_ui()		# Code chunk allows for taking screenshots of levels
+#
+#	yield(get_tree().create_timer(0.5), "timeout")
+#	var img = get_viewport().get_texture().get_data()
+#	img.convert(Image.FORMAT_RGBA8)
+#	img.flip_y()
+#	img.save_png("user://screenshots/%s.png" % level_info["name"].to_lower())
```

Comment:
Is it worth making this a keybind in debug build only? I don't like having commented out code blocks.

Reply:
Now that it's been commented in a commit, I think it's safe to remove this code. If we still need it, we can dig it up from the commit history.

Though if we change levels/add more levels, this code will still be useful...

I'm with Andrew here